src/components/dashboard/SidebarContent.js

```diff
@@ -14,29 +14,33 @@ import FaSignOut from 'react-icons/lib/fa/sign-out';
 class SidebarContent extends Component {
     render() {
         const bottomTextClass = "b dim white";
-        const bottomLinkClass = "no-underline ph2";
+        const bottomLinkClass = "no-underline";
 
         const bloodlines = <BloodlinesSidebar location={this.props.location}/>;
         const roaster = <RoasterSidebar location={this.props.location}/>;
 
+        const isDev = process.env.NODE_ENV !== 'production';
```

Comment:
should probably put this on the router as well to prevent people from going directly to the url either